packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -3,7 +3,8 @@ import PropTypes from 'prop-types';
 import defaultsDeep from 'lodash/defaultsDeep';
 import Badge from '@leafygreen-ui/badge';
 import Button from '@leafygreen-ui/button';
-import Icon from '@leafygreen-ui/icon';
+import ArrowRightIcon from '@leafygreen-ui/icon/dist/ArrowRight';
+import { Size } from '@leafygreen-ui/icon';
```

Comment:
I think we might have to avoid these types of imports still since this is coming from a commonjs module. Otherwise, this may mitigate the benefits we'll get from doing direct imports.

Nothing wrong with using a string below anyways – typechecking will catch any typos.

Reply:
👍